app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::UsersController < ApplicationController
+
+  def index
+    unless user_signed_in? && current_user.admin?
+      redirect_to '/'
+    end
+  end
+
+
+end
```

Comment:
It might be worth it to abstract this into a general `before_action`, something like `authenticate_admin`, if you end up needing it more than once.